python/pyspark/ml/__init__.py

```diff
@@ -15,6 +15,6 @@
 # limitations under the License.
 #
 
-from pyspark.ml.pipeline import Transformer, Estimator, Model, Pipeline, PipelineModel, Evaluator
+from pyspark.ml.pipeline import Transformer, Estimator, Model, Pipeline, PipelineModel
 
-__all__ = ["Transformer", "Estimator", "Model", "Pipeline", "PipelineModel", "Evaluator"]
+__all__ = ["Transformer", "Estimator", "Model", "Pipeline", "PipelineModel"]
```

Comment:
On the Scala side, we put `Classifier` under `ml.classification` and `Regressor` under `ml.regression`. So I moved `Evaluator` to `ml.evaluation` to match them. `Evaluator` is now under `pyspark.ml.evaluation` and hence it is not imported here. Under `pyspark.ml`, we have `Transformer`, `Estimator`, `Model`, `Pipeline`, and `PipelineModel`.